addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -294,7 +294,9 @@ class ChatWindow extends Component {
          * Useful in order to allow copy/paste content inside chat window with
          * CTRL-C & CTRL-V when on the home menu.
          */
-        ev.stopPropagation();
+        if (!ev.altKey) {
+            ev.stopPropagation();
+        }
```

Comment:
Hello, could you try to remove the `stopPropagation` altogether? It shouldn't be needed, and I can't seem to reproduce the fuzzy-search issue that it was supposed to fix.

Reply:
> Hello, could you try to remove the `stopPropagation` altogether? It shouldn't be needed, and I can't seem to reproduce the fuzzy-search issue that it was supposed to fix.

Hello, 'stopPropagation' is Useful in order to allow copy/paste content inside chat window with CTRL-C & CTRL-V when on the home menu.If we remove the 'stopPropagation' and when focus is on chat window it will do fuzzy search.

Reply:
But I think this condition is obsolete. Can you reproduce the ctrl+v issue? Because I can't. I think it was necessary some time in the past but not since the home menu was rewritten in OWL.